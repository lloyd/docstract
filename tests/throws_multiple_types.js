/**
 * a module with a function thats really throwey. 
 */

/**
 * @function throweyFunc
 * @throws When you are sleeping
 * @throws When you're awake
 * @throws {object} if you've been bad or good.
 * @throws So be good for goodness sake.
 */
