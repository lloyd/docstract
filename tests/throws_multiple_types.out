{
  "desc": "a module with a function thats really throwey.", 
  "filename": "tests/throws_multiple_types.js", 
  "functions": [
    {
      "name": "throweyFunc", 
      "source_lines": [
        5, 
        11
      ], 
      "throws": [
        {
          "desc": "When you are sleeping"
        }, 
        {
          "desc": "When you're awake"
        }, 
        {
          "desc": "if you've been bad or good.", 
          "type": "object"
        }, 
        {
          "desc": "So be good for goodness sake."
        }
      ]
    }
  ], 
  "module": "throws_multiple_types", 
  "source_lines": [
    1, 
    4
  ]
}
